client/me/concierge/shared/appointment-info.js

```diff
@@ -145,17 +145,12 @@ class AppointmentInfo extends Component {
 			<div>
 				<Confirmation
 					title={ translate( 'Your upcoming appointment' ) }
-					description={ translate(
-						'We can talk about anything related to your site. ' +
-							'Get all your questions ready ' +
-							'-- we look forward to chatting!',
-						{
-							args: {
-								beginTime: moment( beginTimestamp ).format( beginTimeFormat ),
-								duration: moment( endTimestamp ).diff( beginTimestamp, 'minutes' ),
-							},
-						}
-					) }
+					description={ translate( 'Get all your questions ready — we look forward to chatting!', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'We can talk about anything related to your site. Get all your questions ready -- we look forward to chatting!' )` *ES Score: 7*